packages/flutter/test/scheduler/scheduler_test.dart

```diff
@@ -237,6 +239,22 @@ void main() {
     warmUpDrawFrame();
     expect(scheduler.hasScheduledFrame, isTrue);
   });
+
+  test('The task was successfully executed after the animation', () async {
+    scheduler.schedulingStrategy = defaultSchedulingStrategy;
+    scheduler.resetEventLoop();
```

Comment:
Why do we need to do this? Isn't that the default?

Reply:
yeah, the scheduler.schedulingStrategy has been changed by other test
https://github.com/flutter/flutter/blob/ce889fa883c8d86b64a9fed1462740315045ac1a/packages/flutter/test/scheduler/scheduler_test.dart#L44-L45